### ISSUE TITLE
Delete image from gallery after upload

### ISSUE THREAD

**Issue by neslihanturan** [commons-app/apps-android-commons#4738]
**Summary:** 

We got a request from a user says 

"The Nominate for deletion button is for deleting from Commons.
My request is for deleting uploaded image from my phone, so that phone space is saved once uploading is complete. Presently, I have to manually delete from phone each time."

I am not sure if this feature would be wanted from all users but lets discuss.

**Comment by sivaraam** [comment]
> I am not sure if this feature would be wanted from all users but lets discuss.

I don't think everyone would want this feature either but it might certainly be a nice feature to have for someone who uses a device that has low storage or for someone who just don't want to keep stocking up images on their device. A lot of good quality images could easily eat up a lot of device storage. So, if we ever implement this we could have it as an opt-in feature.

**Comment by Rishavgupta12345** [comment]
@nicolas-raoul  this feature is not useful enough for all the users , and I think we should close this issue

**Comment by nicolas-raoul** [comment]
I think we can keep this as a low priority task. Before deleting pictures we should probably make upload super-reliable.
